mongodbatlas/resource_mongodbatlas_privatelink_endpoint_service_test.go

```diff
@@ -80,10 +80,11 @@ func TestAccResourceMongoDBAtlasPrivateLinkEndpointServiceAWS_import(t *testing.
 				),
 			},
 			{
-				ResourceName:      resourceName,
-				ImportStateIdFunc: testAccCheckMongoDBAtlasPrivateLinkEndpointServiceImportStateIDFunc(resourceName),
-				ImportState:       true,
-				ImportStateVerify: true,
+				ResourceName:            resourceName,
+				ImportStateIdFunc:       testAccCheckMongoDBAtlasPrivateLinkEndpointServiceImportStateIDFunc(resourceName),
```

Comment:
I have a question, is it necessary to use so long func names?

I mean, in this method you are in `mongodbatlas_privatelink_endpoint...`method, maybe the name for this func can be `testAccCheckServiceImportStateIDFunc`

having names like this can affect the readability and the maintainability

Reply:
As long it doesn't cross the limit of the line-length from golangci.yml it should be fine, mm it seems readable for me since it's usually testAccCheckMongoDBAtlas[Resource/datasource][Extra(like ImportState, destroy, etc] something like that, but it somehow it crosses the limit, yes it would be nice to shorten it and still readable for the resource it belongs to